pkg/network/namescheme/networknamescheme.go

```diff
@@ -92,17 +92,15 @@ func CreateOrdinalNetworkNameScheme(vmiNetworks []v1.Network) map[string]string
 	return networkNameSchemeMap
 }
 
+// OrdinalPodInterfaceName returns the ordinal interface name for the given network name.
+// Networks contains pod and multus networks, so create a corresponding map then compare the target name.
```

Comment:
This is an implemention detail I woudnt expose as it may change, as now :slightly_smiling_face: 
Also, in the context of this function you dont know what will the networks slice consist of.